package.json

```diff
@@ -8,14 +8,16 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "web-vitals": "^1.1.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "cypress": "cypress: open"
```

Comment:
Suggested correction: the value of the `"cypress"` script should read `"cypress open"`.  The extra colon may throw an error.  Please run cypress to confirm.